requirements.txt

```diff
@@ -1 +1,5 @@
 autotest>=0.16.2; python_version < '3.0'
+aexpect>1.5.0
+simplejson>=3.5.3
+netaddr>=0.7.18
+netifaces>=0.10.5
```

Comment:
Would you please clarify why these things are added back?

Reply:
Because `python setup.py install` won't install the dependencies (unlike `pip install .`)

Reply:
... so for py3 it shouldn't matter as these deps will be already installed by `pip install` and on py2 it'll define all the required deps.

Reply:
> Because `python setup.py install` won't install the dependencies (unlike `pip install .`)

@ldoktor seems not true on my side, would you please help confirm that?
```
# python setup.py install
running install
running bdist_egg
running egg_info
creating avocado_framework_plugins_vt.egg-info
writing requirements to avocado_framework_plugins_vt.egg-info/requires.txt
writing avocado_framework_plugins_vt.egg-info/PKG-INFO
writing top-level names to avocado_framework_plugins_vt.egg-info/top_level.txt
...
Adding avocado-framework-plugins-vt 71.0 to easy-install.pth file

Installed /tmp/test/lib/python2.7/site-packages/avocado_framework_plugins_vt-71.0-py2.7.egg
Processing dependencies for avocado-framework-plugins-vt==71.0
Searching for avocado-framework>=68.0
Reading https://pypi.org/simple/avocado-framework/
...
Adding avocado-framework 71.0 to easy-install.pth file
Installing avocado script to /tmp/test/bin

Installed /tmp/test/lib/python2.7/site-packages/avocado_framework-71.0-py2.7.egg
Searching for six
Reading https://pypi.org/simple/six/
...
Downloading https://files.pythonhosted.org/packages/e3/24/c35fb1c1c315fc0fffe61ea00d3f88e85469004713dab488dee4f35b0aff/simplejson-3.16.0.tar.gz#sha256=b1f329139ba647a9548aa05fb95d046b4a677643070dc2afc05fa2e975d09ca5
Best match: simplejson 3.16.0
Processing simplejson-3.16.0.tar.gz
Writing /tmp/easy_install-kq59PM/simplejson-3.16.0/setup.cfg
...
Installed /tmp/test/lib/python2.7/site-packages/aexpect-1.5.1-py2.7.egg
Searching for netifaces
Reading https://pypi.org/simple/netifaces/
Downloading https://files.pythonhosted.org/packages/7e/02/ad1a92a72620cc17d448fe4dbdfbdf8fe1487ee7bfd82bb48308712c2f3c/netifaces-0.10.9-cp27-cp27mu-manylinux1_x86_64.whl#sha256=4921ed406386246b84465950d15a4f63480c1458b0979c272364054b29d73084
Installed /tmp/test/lib/python2.7/site-packages/aexpect-1.5.1-py2.7.egg
Searching for netifaces
Reading https://pypi.org/simple/netifaces/
Downloading https://files.pythonhosted.org/packages/7e/02/ad1a92a72620cc17d448fe4dbdfbdf8fe1487ee7bfd82bb48308712c2f3c/netifaces-0.10.9-cp27-cp27mu-manylinux1_x86_64.whl#sha256=4921ed406386246b84465950d15a4f63480c1458b0979c272364054b29d73084
...
Installed /tmp/test/lib/python2.7/site-packages/subprocess32-3.5.4-py2.7-linux-x86_64.egg
Searching for setuptools==41.2.0
Best match: setuptools 41.2.0
Adding setuptools 41.2.0 to easy-install.pth file
Installing easy_install script to /tmp/test/bin
Installing easy_install-3.6 script to /tmp/test/bin

Using /tmp/test/lib/python2.7/site-packages
```

Reply:
I don't remember which system I used, but I'll try it again next week.

Reply:
makes sense to me, thanks.